src/main/scala/org/tensorframes/impl/DebugRowOps.scala

```diff
@@ -909,7 +909,7 @@ object DebugRowOpsImpl extends Logging {
   private def performRunner(
       session: Session,
       requested: Seq[String],
-      inputs: Seq[(String, Tensor)]): Seq[Tensor] = {
+      inputs: Seq[(String, Tensor[_])]): Seq[Tensor[_]] = {
```

Comment:
can you instead change the signature to 

```scala
def performRunner[T](..., Tensor[T]): Seq[Tensor[T]] = ...
```
It is more precise.

Reply:
sure, will do. I thought the meaning was the same though since T is not not bounded to anything?

Reply:
Oh actually it seems I did misunderstand the meaning of _. Anyways, I tried to replace it with T but hit issues. To begin with, are the input tensors strictly required to be of the same type? I looked over the code a bit but it was not clear to me if tat was the case.